docs/sources/tempo/configuration/parquet.md

```diff
@@ -28,18 +28,18 @@ It introduces dedicated attribute columns, which improve query performance by st
 rather than in the generic attribute key-value list.
 For more information, see [Dedicated attribute columns]({{< relref "../operations/tempo_cli" >}}).
 
-You can still use the previous format `vParquet2`.
-To enable it, set the block version option to `vParquet2` in the Storage section of the configuration file.
+You can also use the latest more experimental version `vParquet4`.
+To enable it, set the block version option to `vParquet4` in the Storage section of the configuration file.
 
 ```yaml
-# block format version. options: v2, vParquet2, vParquet3
-[version: vParquet2]
+# block format version. options: v2, vParquet2, vParquet3, vParquet4
+[version: vParquet4]
 ```
 
 In some cases, you may choose to disable Parquet and use the old `v2` block format. Using the `v2` block format disables all forms of search, but also reduces resource consumption, and may be desired for a high-throughput cluster that does not need these capabilities. To make this change, set the block version option to `v2` in the Storage section of the configuration file.
```

Comment:
Do people still have the option of disabling Parquet in 2.5?

Reply:
Yes, it's still an option